System.IO.Abstractions.TestingHelpers.Tests/MockDriveInfoTests.cs

```diff
@@ -59,13 +59,32 @@ public void MockDriveInfo_RootDirectory_ShouldReturnTheDirectoryBase()
             var fileSystem = new MockFileSystem();
             fileSystem.AddDirectory(XFS.Path(@"c:\Test"));
             var driveInfo = new MockDriveInfo(fileSystem, "c:");
-            var expectedDirectory = XFS.Path(@"C:\");
+            var expectedDirectory = XFS.Path(@"c:\");
 
             // Act
             var actualDirectory = driveInfo.RootDirectory;
 
             // Assert
             Assert.AreEqual(expectedDirectory, actualDirectory.FullName);
         }
+
+        [TestCase("c:","c:\\")]
+        [TestCase("C:","C:\\")]
+        [TestCase("d:","d:\\")]
+        [TestCase("e:","e:\\")]
+        [TestCase("f:","f:\\")]
+        public void MockDriveInfo_ToString_ShouldReturnTheDrivePath(string path, string expectedPath)
+        {
+            // Arrange
+            var directoryPath = XFS.Path(path);
+            
+            // Act
+            var mockDriveInfo = new MockDriveInfo(new MockFileSystem(), directoryPath);
+            var realDriveInfo = new DriveInfo(directoryPath);
```

Comment:
We usually don't "compare" against real implementations, so please remove this one

Reply:
Removed real implementation in test